app/controllers/dashboards_controller.rb

```diff
@@ -27,15 +26,8 @@ def dashboard
   end
 
   def set_defence_requests
-    if @dashboard.user_role == "solicitor"
-      @active_defence_requests = @dashboard.defence_requests.active
-    else
-      @active_defence_requests = @dashboard.defence_requests
-    end
+    @active_defence_requests = @dashboard.defence_requests.active
     @closed_defence_requests = @dashboard.defence_requests.finished
```

Comment:
We always load both sets of defence requests, even though only one is used. Same for the counts bellow.